app/assets/javascripts/annotations.js

```diff
@@ -342,7 +342,11 @@ function attachEvents() {
 function attachChangeFileEvents() {
   // Set up file switching to use the local cache
   function changeFileClickHandler(e) {
-    const wasCachedLocally = changeFile($(this).data("header_position"));
+    const headerURL = new URLSearchParams(location.search);
+    const currentHeader = parseInt(headerURL.get("header_position"));
```

Comment:
nit: no need for this; there's already a `currentHeaderPos` global variable in annotations.js that will give you this information